src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/createaccessrequirement/CreateManagedACTAccessRequirementStep3.java

```diff
@@ -0,0 +1,213 @@
+package org.sagebionetworks.web.client.widget.accessrequirements.createaccessrequirement;
+
+import static com.google.common.util.concurrent.MoreExecutors.directExecutor;
+
+import java.util.HashSet;
+
+import org.sagebionetworks.repo.model.AccessControlList;
+import org.sagebionetworks.repo.model.ManagedACTAccessRequirement;
+import org.sagebionetworks.repo.model.ResourceAccess;
+import org.sagebionetworks.repo.model.principal.TypeFilter;
+import org.sagebionetworks.web.client.DisplayConstants;
+import org.sagebionetworks.web.client.SynapseJavascriptClient;
+import org.sagebionetworks.web.client.utils.CallbackP;
+import org.sagebionetworks.web.client.widget.search.SynapseSuggestBox;
+import org.sagebionetworks.web.client.widget.search.UserGroupSuggestion;
+import org.sagebionetworks.web.client.widget.search.UserGroupSuggestionProvider;
+import org.sagebionetworks.web.client.widget.table.modal.wizard.ModalPage;
+import org.sagebionetworks.web.client.widget.team.UserTeamBadge;
+import org.sagebionetworks.web.shared.exceptions.NotFoundException;
+import org.sagebionetworks.web.shared.users.AclUtils;
+import org.sagebionetworks.web.shared.users.PermissionLevel;
+
+import com.google.common.util.concurrent.FluentFuture;
+import com.google.common.util.concurrent.FutureCallback;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+
+/**
+ * Second page of creating an access requirement (ACT)
+ * 
+ * @author Jay
+ *
+ */
+public class CreateManagedACTAccessRequirementStep3 implements ModalPage, CreateManagedACTAccessRequirementStep3View.Presenter {
+	CreateManagedACTAccessRequirementStep3View view;
+	ModalPresenter modalPresenter;
+	ManagedACTAccessRequirement accessRequirement;
+	SynapseJavascriptClient jsClient;
+	AccessControlList originalAcl;
+	String reviewerPrincipalId;
+	UserTeamBadge reviewerUserTeamBadge;
```

Comment:
Confirmed with Marco that ACT always has reviewer access (and does not need to be represented in the ACL).  This UI is to support a single entry in the ACL (where the principal has the REVIEW_SUBMISSIONS access type).